File: linear_search.js
function linear_search(list, target) {
    for (let i = 0; i < list.length; i++) {
        if (list[i] === target)
            return i
    }
    return null;
}

function verify(index) {
    if (index !== null) {
        console.log("Target found at index", index)
    } else console.log("Target not found in list")

}

let numbers = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]

const result = linear_search(numbers, 17)
verify(result)

File: linear_search.rb
def linear_search(list, target)
    for i in 0..(list.length)
        if list[i] == target
            return i
        end
    end
    return nil;
end

def verify (index)
    if index != nil
        puts "Target found at index, #{index}"
    else
        puts "Target not in the list"
    end
end

numbers = [1,2,3,4,5,6,7,8,9,10]

result = linear_search(numbers, 10)
verify(result)